---
layout: paper
title: "Metagenomic global survey and in-depth genomic analyses of Ruminococcus gnavus reveal differences across host lifestyle and health status"
nickname: 2025-01-30-nooij-metagenomic-global-survey
authors: "Nooij S, Plomp N, Sanders IMJG, Schout L, van der Meulen AE, Terveer EM, Norman JM, Karcher N, Larralde MF, Vossen RHAM, Kloet SL, Faber KN, Harmsen HJM, Zeller GF, Kuijper EJ, Smits WK, Ducarmon QR"
year: "2025"
journal: "Nat Commun"
volume: 16
issue: 1
pages: 1182
is_published: true
image: /assets/images/papers/nat-commun.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: PMC11782615
preprint: https://www.biorxiv.org/content/10.1101/2024.06.27.600998v1
supplement:

# Links
doi: "10.1038/s41467-025-56449-x"
pmid: 39885121

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

Ruminococcus gnavus is a gut bacterium found in > 90% of healthy individuals, but its increased abundance is also associated with chronic inflammatory diseases, particularly Crohn's disease. Nevertheless, its global distribution and intraspecies genomic variation remain understudied. By surveying 12,791 gut metagenomes, we recapitulated known associations with metabolic diseases and inflammatory bowel disease. We uncovered a higher prevalence and abundance of R. gnavus in Westernized populations and observed bacterial relative abundances up to 83% in newborns. Next, we built a resource of R. gnavus isolates (N = 45) from healthy individuals and Crohn's disease patients and generated complete R. gnavus genomes using PacBio circular consensus sequencing. Analysis of these genomes and publicly available high-quality draft genomes (N = 333 genomes) revealed multiple clades which separated Crohn's-derived isolates from healthy-derived isolates. Presumed R. gnavus virulence factors could not explain this separation. Bacterial genome-wide association study revealed that Crohn's-derived isolates were enriched in genes related to mobile elements and mucin foraging. Together, we present a large R. gnavus resource that will be available to the scientific community and provide novel biological insights into the global distribution and genomic variation of R. gnavus.
