---
layout: paper
title: "Development of a GC-MS/MS method to quantify 120 gut microbiota-derived metabolites"
nickname: {nickname}
authors: "Denisov N, Springer F, Brauer-Nikonow A, Maftei G, Zeller G, Medeiros Selegato D, Zimmermann M"
year: "2025"
journal: "ChemRxiv"
volume:
issue:
pages:
is_published: false
image: /assets/images/papers/chemrxiv.png
projects:
tags: ["preprint"]

# Text
fulltext:
pdf:
pdflink:
pmcid:
preprint: "https://doi.org/10.26434/chemrxiv-2025-gwggz"
supplement:

# Links
doi: "10.26434/chemrxiv-2025-gwggz"
pmid:

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{{% include JB/setup %}}

# Abstract

The gut microbiota produces metabolites that are important for host physiology and have critical roles in the development of diseases, such as metabolic disorders, cardiovascular diseases, and cancer. Here, we developed a gas chromatography-coupled to tandem mass spectrometry (GC-MS/MS) method for the quantification of 120 volatile and semi-volatile compounds produced by gut bacteria, including short-chain fatty acids, indols, nucleotides, organic acids, and amino acid derivatives. The method is based on multiple-reaction-monitoring (MRM) of each analyte and their respective isotopically labeled internal standard, enabling absolute metabolite quantification between 0.45 pmol and 1 nmol. Applying the method to different tissue samples from germfree and conventionally colonized mice, we illustrate the ability to quantify microbiota-produced metabolites in different sample matrices â€“ plasma, liver, feces, and intestinal content, and at different concentrations. Lastly, we demonstrate that this protocol is capable of quantifying microbiota-derived metabolites in stool samples stored in DNA stabilization buffers that are typically used in sequencing-based microbiome studies. Altogether, the developed GC-MS/MS method adds a valuable analytical tool to quantify microbiota-host metabolic interactions.
