---
layout: tools
title: "PySkANI"
contributors: [mlarralde,carroll]
handle: pyskani
status: complete
type: software

# Optional
website: "https://github.com/althonos/pyorthoani"
publications: "https://academic.oup.com/nargab/article/7/3/lqaf095/8196481"
doi: "10.1093/nargab/lqaf095"
image: /assets/images/tools/2025-07-11-pyskani-icon.png
tagline: PyO3 bindings and Python interface to skani, a method for fast genomic identity calculation using sparse chaining. 
tags: [bioinformatics, ANI]

# Data and code
github: ["https://github.com/althonos/pyorthoani"]
---
{% include JB/setup %}


## Abstract

Average Nucleotide Identity through sparse k-mer chaining (skani) is a method for high-throughput computation of
ANI without alignments. It uses a *sparse* k-mer seeding method aiming at reducing the set of k-mers needed
to represent a genome, thus allowing for high mapping speeds. In addition, the *chaining* method can be 
used to compute a partial sequence mapping, which generates results of higher quality than pure k-mer
estimation techniques.

PySkANI is a [Python](https://python.org) package which provides binding to the skani code using [PyO3](https://pyo3.rs).
It implements an object-oriented interface to manipulate skani databases, and query them for ANI containment from 
Python code.

Pyskani was developed as part of our Python ANI suite, in collaboration with [The CompMicroLab at Ume√• University](https://www.microbe.dev/): 
[PyOrthoANI, PyFastANI, and Pyskani: a suite of Python libraries for computation of average nucleotide identity](https://doi.org/10.1093/nargab/lqaf095).
